components/core/src/os/process/windows.rs

```diff
@@ -47,3 +50,147 @@ fn become_child_command(command: PathBuf, args: Vec<OsString>) -> Result<()> {
     // Let's honor the exit codes from the child process we finished running
     process::exit(status.code().unwrap())
 }
+
+fn handle_from_pid(pid: u32) -> Option<winapi::HANDLE> {
+    unsafe {
+        let proc_handle = kernel32::OpenProcess(
+            winapi::PROCESS_QUERY_LIMITED_INFORMATION,
+            winapi::FALSE,
+            pid as winapi::DWORD
+        );
+
+        // we expect this to happen if the process died
+        // before OpenProcess completes
+        if proc_handle == ptr::null_mut() {
+            return None;
+        }
+        else {
+            return Some(proc_handle);
+        }
+    }
+}
+
+pub struct Child {
+    handle: Option<winapi::HANDLE>,
+    last_status: Option<u32>,
```

Comment:
It is but here is the [signature](https://msdn.microsoft.com/en-us/library/windows/desktop/ms683189(v=vs.85).aspx) for GetExitCodeProcess. Its a pointer to a DWORD.